sound/soc/sof/intel/byt.c

```diff
@@ -483,6 +483,34 @@ static int tangier_pci_probe(struct snd_sof_dev *sdev)
 	return ret;
 }
 
+static int byt_suspend(struct snd_sof_dev *sdev)
```

Comment:
This is inside a `#if IS_ENABLED(CONFIG_SND_SOC_SOF_MERRIFIELD)` block which fails to build baytail fi merrifield is not enabled